tests.sh

```diff
@@ -29,7 +29,7 @@ function set_default(){
 # Set undefined/default test params
 set_default NUMBER_OF_POOLS 1
 set_default no_of_ps $(($DAOS_CLIENTS * $PPC))
-set_default CONT_RF 0
```

Comment:
Can we please also keep `set_default CONT_RF 0`? That way we can put the default value of 0 in the database without the database itself having to set a default

Reply:
Ah sure